backend/spec/rails_helper.rb

```diff
@@ -0,0 +1,64 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
👀 I think I'll need to familiarize myself with this at some point

Reply:
Yeah, `Rspec` is one of the test frameworks available to Rails, and it's the one I've been using professionally for nearly 8 years now. I really like the way it's written and the matchers available for testing.

This file is like a startup script that provides all the necessary methods and modules for Rspec tests to run. We don't modify this very often, unless we want to make new helper methods that are available to every Rspec test.